backend/pkg/api/data_access/vdb_helpers.go

```diff
@@ -211,10 +213,15 @@ func (d *DataAccessService) getElClAPR(ctx context.Context, dashboardId t.VDBId,
 		aprDivisor = 90 * 24
 	}
 
-	// invested amount is not post-pectra safe
-	investedAmount := d.convertClToMain(decimal.NewFromUint64(d.config.ClConfig.MaxEffectiveBalance))
+	investedAmountUInt, err := d.GetValidatorDashboardEffectiveBalanceTotal(ctx, dashboardId, true)
+	if err != nil {
+		return decimal.Zero, 0, decimal.Zero, 0, fmt.Errorf("error retrieving total effective balance: %w", err)
```

Comment:
Thought: we should maybe just return an internal struct here instead of 4 values + 1 error